Tests/LinuxMain.swift

```diff
@@ -1,7 +1,7 @@
-import XCTest
-
-import RxCBCentralTests
-
-var tests = [XCTestCaseEntry]()
-tests += RxCBCentralTests.allTests()
-XCTMain(tests)
+//import XCTest
+//
+//import RxCBCentralTests
+//
+//var tests = [XCTestCaseEntry]()
+//tests += RxCBCentralTests.allTests()
+//XCTMain(tests)
```

Comment:
What's your reasoning for commenting these lines out?

Reply:
Oops, sorry! I commented them because my xcode won't build with it - but it wasn't supposed to be in the commit.